### PR TITLE
[DVCSMP-3682] On-boarding Fibaro Double Switch 2

### PR DESCRIPTION
**Pull request by sagar-trility**

### CODE REVIEW

**Comment by sagar-trility** [comment]
@greens This is ready for your review. Thank you for your help.

File: devicetypes/fibargroup/fibaro-double-switch-2-usb.src/fibaro-double-switch-2-usb.groovy [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+metadata {
+    definition (name: "Fibaro Double Switch 2 - USB", namespace: "FibarGroup", author: "Fibar Group") {
+        capability "Switch"
+        capability "Actuator"
+        capability "Sensor"
+        capability "Energy Meter"
+        capability "Power Meter"
+
+        command "reset"
+        command "refresh"
```
Comment (greens) [inline devicetypes/fibargroup/fibaro-double-switch-2-usb.src/fibaro-double-switch-2-usb.groovy]: Replace with "Refresh" capability.

File: devicetypes/fibargroup/fibaro-double-switch-2-zw5.src/fibaro-double-switch-2-zw5.groovy [hunk 2]
```diff
@@ -0,0 +1,639 @@
...
+    definition (name: "Fibaro Double Switch 2 ZW5", namespace: "FibarGroup", author: "Fibar Group") {
+        capability "Switch"
+        capability "Energy Meter"
+        capability "Power Meter"
+        capability "Button"
+        capability "Configuration"
+        capability "Health Check"
+
+        command "reset"
+        command "refresh"
```
Comment (greens) [inline devicetypes/fibargroup/fibaro-double-switch-2-zw5.src/fibaro-double-switch-2-zw5.groovy]: same as above

File: devicetypes/fibargroup/fibaro-double-switch-2-zw5.src/fibaro-double-switch-2-zw5.groovy [hunk 3]
```diff
@@ -0,0 +1,639 @@
...
+        fingerprint mfr: "010F", prod: "0203", model: "2000"
+        fingerprint deviceId: "0x1001", inClusters:"0x5E,0x86,0x72,0x59,0x73,0x22,0x56,0x32,0x71,0x98,0x7A,0x25,0x5A,0x85,0x70,0x8E,0x60,0x75,0x5B"
+        fingerprint deviceId: "0x1001", inClusters:"0x5E,0x86,0x72,0x59,0x73,0x22,0x56,0x32,0x71,0x7A,0x25,0x5A,0x85,0x70,0x8E,0x60,0x75,0x5B"
```
Comment (greens) [inline devicetypes/fibargroup/fibaro-double-switch-2-zw5.src/fibaro-double-switch-2-zw5.groovy]: This fingerprint would also match a device with the CCs on the line above. 

Generally, this sort of fingerprinting is intended as a catch-all for devices with generic functionality. A hyper-specific fingerprint like this doesn't serve that utility. I'd be more comfortable just leaving the manufacturer-specific fingerprint.

File: devicetypes/fibargroup/fibaro-double-switch-2-usb.src/fibaro-double-switch-2-usb.groovy (hunk 1 above)
Comment (sagar-trility) [reply devicetypes/fibargroup/fibaro-double-switch-2-usb.src/fibaro-double-switch-2-usb.groovy]: @greens This change has been made.

File: devicetypes/fibargroup/fibaro-double-switch-2-zw5.src/fibaro-double-switch-2-zw5.groovy (hunk 2 above)
Comment (sagar-trility) [reply devicetypes/fibargroup/fibaro-double-switch-2-zw5.src/fibaro-double-switch-2-zw5.groovy]: This change has been made.

File: devicetypes/fibargroup/fibaro-double-switch-2-zw5.src/fibaro-double-switch-2-zw5.groovy (hunk 3 above)
Comment (sagar-trility) [reply devicetypes/fibargroup/fibaro-double-switch-2-zw5.src/fibaro-double-switch-2-zw5.groovy]: I have made changes just to have the mfr, prod and model number.

**Review by greens — CHANGES_REQUESTED** [review]
Actually there's more places than the ones I found. Can you convert the spacing to tabs and then make sure it's consistent?

**Comment by sagar-trility** [comment]
This PR is closed. I will create a new PR.

**Pull request closed without merging**